app/routes/events/view/settings.js

```diff
@@ -11,5 +11,13 @@ export default Route.extend({
       roleInvites : await eventDetails.query('roleInvites', {}),
       roles       : await this.store.findAll('role')
     };
+  },
+  afterModel(model) {
```

Comment:
Is this block necessary? @shreyanshdwivedi

Reply:
@kushthedude Yes. An `afterModel` hook is used for such transitions and this is why the PR was actually proposed. Reading the docs would be helpful https://guides.emberjs.com/release/routing/redirection/

Reply:
@shreyanshdwivedi Yeah, so I thought that you required the owner from the model. But as that isn't needed and you can directly use this.authManager.user, beforeModel would do